app/src/main/java/ch/epfl/sdp/blindwar/tutorial/DemoActivity.kt

```diff
@@ -0,0 +1,88 @@
+package ch.epfl.sdp.blindwar.tutorial
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.EditText
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import ch.epfl.sdp.blindwar.R
+import ch.epfl.sdp.blindwar.game.GameTutorial
+import ch.epfl.sdp.blindwar.game.MusicMetaData
+import com.chibde.visualizer.CircleVisualizer
+import com.chibde.visualizer.LineVisualizer
+
+class DemoActivity: AppCompatActivity() {
+    private lateinit var game: GameTutorial
+    private var playing = true
+    private lateinit var guessEditText: EditText
+    private lateinit var scoreTextView: TextView
+    private lateinit var musicMetaData: MusicMetaData
+    private lateinit var visualizer: CircleVisualizer
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_demo)
+
+        // Create the game instance with difficulty easy
+        this.game = GameTutorial(assets, 5000)

```

Comment:
It should be more clear that 5000 corresponds to the difficulty easy. Difficulties should be better managed/more clear in the code if it's supposed to be a real feature.

Reply:
I have planned to  do that while working on the solo mode ! But good remark !